content/applications/hr/time_off/management.rst

```diff
@@ -0,0 +1,150 @@
+be used to narrow down the presented time off requests, located beneath the :guilabel:`Status` and
+:guilabel:`Department` headings.
+
+Since only time off requests that need to be approved are shown, the only :guilabel:`status` options
+are :guilabel:`All`, :guilabel:`To Approve`, and :guilabel:`Second Approval`.
+
+The various departments the user is a member of, and manages employees under, also appear on the
+left side of the page, under :guilabel:`Departments`.
+
+.. note::
+   If there are no requests that fall under one of the status options or departments, that status
+   or department is **not** visible on the left-side menu.
+
+To only display time off requests for specific departments, click on the :guilabel:`Department` on
+the left-hand side of the page. Only requests within the selected department are then presented.
+
+The status column displays the status of each request, with the status highlighted in a specific
+color.
+
+The :guilabel:`To Approve` and :guilabel:`Second Approval` requests are highlighted in yellow, and
+are the only ones that appear in the list by default.
+
+If the `Waiting For Me` filter is removed, all statuses appear.
+
+:guilabel:`Approved` requests are highlighted in green, :guilabel:`To Submit` (drafts) requests are
+highlighted in blue, and the :guilabel:`Refused` requests are highlighted in gray.
+
+To approve a time off request, click the :icon:`fa-thumbs-up` :guilabel:`Approve` button at the end
+of the line.
+
+To validate a time off request that has already been approved, and is waiting on a second approval,
+click the :icon:`fa-check` :guilabel:`Validate` button at the end of the line.
+
+To refuse a request, click the :icon:`fa-times` :guilabel:`Refuse` button at the far end of the
+line.
+
+.. image:: management/time-off-requests.png
+   :align: center
+   :alt: Time off requests with the filter, groupings, and status sections highlighted.
+
+For more details, click anywhere on the time off request line (except for the :icon:`fa-thumbs-up`
+:guilabel:`Approve`, :icon:`fa-check` :guilabel:`Validate` icon, and :icon:`fa-times`
+:guilabel:`Refuse` buttons). Doing so loads that specific time off request form. Depending on the
+rights of the user, changes can be made.
+
+To modify the request, make any desired changes to the form. All changes are automatically saved.
+
+It is also possible to approve, validate, or refuse the request from this form. Click the
+:guilabel:`Approve` button to approve, the :guilabel:`Validate` button to validate, or the
+:guilabel:`Refuse` button to refuse the request.
+
+.. _time_off/manage-allocations:
+
+Manage allocations
+==================
+
+To view allocations that need approval, navigate to :menuselection:`Time Off app --> Management -->
+Allocations`. Doing so reveals the :guilabel:`Allocations` page.
+
+The only allocations visible on this page are for employees the user either has :guilabel:`Time Off
```

Comment:
I'm confused on the subject here, is it "employees" or "the user?"
Consider picking one, for clarity.